docs/design/autoware-interfaces/ad-api/features/interface.md

```diff
@@ -0,0 +1,17 @@
+# Interface feature
```

Comment:
[nits][IMO] I feel replacing `feature` with `API` looks better.

```suggestion
# Interface API
```

Reply:
This page focuses on features. The API is defined to provide these features and is described on a separate page.
Related comment is https://github.com/autowarefoundation/autoware-documentation/pull/115#discussion_r886559084.